Mocking/src/test/java/com/gepardec/tdd/workdir/BookBusinessImplMockInjectionTest.java

```diff
@@ -0,0 +1,40 @@
+package com.gepardec.tdd.workdir;
+
+import com.gepardec.tdd.BookBusinessImpl;
+import com.gepardec.tdd.BookService;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.*;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class BookBusinessImplMockInjectionTest {
+
+    @Test
+    public void usingMockito() {
+        // TODO
```

Comment:
Die //TODO Kommentare könnte man in eine kleine JavaDoc ausbauen, die erklärt, was der Test am Ende machen soll.

Oder man könnte auch die assertions drin lassen und den Test so failen lassen - dann muss man nur noch das mocking selber machen